package.json

```diff
@@ -14,7 +14,6 @@
   "scripts": {
     "build": "webpack",
     "build:development": "NODE_ENV=development webpack",
-    "build:staging": "NODE_ENV=staging webpack",
```

Comment:
fair enough - was wondering if that day way here yet, but its easy enough to undo this then :)